mixer/pkg/runtime/config/ephemeral.go

```diff
@@ -355,12 +355,12 @@ func (e *Ephemeral) processDynamicInstanceConfigs(ctx context.Context, templates
 		var params map[string]interface{}
 		var err error
 		if inst.Params != nil {
-			if _, ok := inst.Params.(map[string]interface{}); !ok {
+			if params, err = toDictionary(inst.Params); err != nil {
```

Comment:
I'm assuming `processDynamicInstanceConfigs` is (in)directly called at config ingestion time in mixer and not in the data path.

Reply:
yes